megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -3983,31 +4038,63 @@ private TreeMap<Integer, Vector<Integer>> getDroppedUnits() {
         Set<Integer> alreadyDropped = cmd.getDroppedUnits();
         // cycle through the bays
         int bayNum = 1;
-        Vector<Bay> Bays = ce.getTransportBays();
-        for (int i = 0; i < Bays.size(); i++) {
-            Bay currentBay = Bays.elementAt(i);
+        Vector<Transporter> transporters = ce.getTransports();
+        List<Entity> currentUnits;
+
+        for (int i = 0; i < transporters.size(); i++) {
+            Transporter currentT = transporters.elementAt(i);
```

Comment:
Please avoid single letter abbreviations. currentT and iTransporter -> currentTransport and infantryTransporter. If this is a length issue then at least currentTrsp and infTransporter or something.